README.md

```diff
@@ -6,13 +6,100 @@ via a dracut module.
+mode. You'll have to make sure you download the appropriate related
+image for the mode you'd like to use. It is a good idea to download
+them before doing the install as the artifacts server that is being
+used currently has very slow download speeds.
+
+For example download:
+
+- [fedora-coreos-29.731.iso](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731.iso)
+
+and one of the following two:
+
+- [fedora-coreos-29.731-metal-bios.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-bios.raw.gz)
+- [fedora-coreos-29.731-metal-uefi.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-uefi.raw.gz)
+
+**NOTE** Those links will quickly become broken because we prune
+         builds. You'll have to find your own links from the 
+         [latest directory](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+#### Test an ISO based install
+
+You can test an install on a bare metal machine by burning the ISO to
+disk and booting it or using ISO redirection via a LOM interface.
+Alternatively you can use a VM like so:
+
+```
+virt-install --name cdrom --ram 4500 --vcpus 2 --disk size=20 --accelerate --cdrom /path/to/fedora-coreos-29.731.iso --network default
+```
+
+**NOTE** To test UEFI boot add `--boot uefi` to the CLI call
+
+One you have booted you will see a screen press `<TAB>` (isolinux) or
+`e` (grub) to edit the kernel command line. Add the parameters to the
+kernel command line telling it what you want it to do. For example:
+
+- `coreos.inst.install_dev=sda`
+- `coreos.inst.image_url=http://example.com/fedora-coreos-29.731-metal-bios.raw.gz`
+- `coreos.inst.ignition_url=http://example.com/config.ign`
+
+**NOTE** make sure to use a `metal-uefi` image if booting via UEFI
+
+Now press `<ENTER>` (isolinux) or `<CTRL-x>` (grub) to kick off the
+install. The install will occur on tty2 and there are very few good
+log statements or debug opportunities. Thee user experience here
+needs work and is tracked in https://github.com/coreos/coreos-installer/issues/5 .
+
+The install should progress and eventually reboot the machine. After
+reboot the machine will boot into the installed system and the
+embedded ignition config should run on first boot.
+
+#### Test a PXE based install
+
+Using the ISO images you can also do a PXE based install. You can
+mount up the ISO images and use the `initramfs.img` and `vmlinuz`
+for PXE boot. Here is an example `pxelinux.cfg` that I used to perform
+a PXE boot:
+
+```
+DEFAULT pxeboot
+TIMEOUT 20
+PROMPT 0
+LABEL pxeboot
+    KERNEL fedora-coreos-29.731.iso/images/vmlinuz
+    APPEND initrd=fedora-coreos-29.731.iso/images/initramfs.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://192.168.1.101:8000/fedora-coreos-29.731-metal-bios.raw.gz coreos.inst.ignition_url=http://192.168.1.101:8000/config.ign
```

Comment:
remove the first `console=tty0` since `console=ttyS0` will overwrite it?

Reply:
I will remove the 2nd option but do note that kernel mesages will go to every `console=` device on the command line. The last `console=` device will get set as the primary console.

Reply:
Today I learned, I'd keep it then.